web/src/main/java/com/futurewei/alcor/web/entity/NodeInfo.java

```diff
@@ -83,6 +86,16 @@ public NodeInfo(String nodeId, String nodeName, String ipAddress, String macAddr
         this.gRPCServerPort = 0;
     }
 
+    public NodeInfo(String id, String name, String localIp, String macAddress, String veth, int gRPCServerPort, String host_dvr_mac) {
```

Comment:
As this is the latest constructor with full parameters, it contains the full implementation while existing constructors refer to this new one.

Reply:
I have a fix about this part ~ but I am not sure if it is correct ~

Reply:
Talked offline and addressed this comment.